app/src/test/java/com/example/habitsmasher/PasswordEncryptTest.java

```diff
@@ -0,0 +1,22 @@
+package com.example.habitsmasher;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class PasswordEncryptTest {
+
+    private PasswordEncrypt _passwordEncrypt;
+    @Before
+    public void setUp(){
+        _passwordEncrypt = new PasswordEncrypt();
+    }
+
+    @Test
+    public void encrypt_password_expectTrue() {
+        String password = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890-=!@#$%^&*()_+[]{}|;:',.<>/?";
+        String encryptedPassword = _passwordEncrypt.encrypt(password);
+        assertEquals("HIJKLMNOPQRSTUVWXYZ[\\]^_`ahijklmnopqrstuvwxyz{|}~qrs89:;<=>?@74D(G*+,e-1/0f2bdtvuBA.35CE6F", encryptedPassword);
```

Comment:
Can we extract the two password strings to constants and name them accordingly so they are more readable? Something like `ORIGINAL_PASSWORD` and `ENCRYPTED_PASSWORD`.

- [ ] extract constants